lib/worker-pipeline.js

```diff
@@ -69,6 +70,29 @@ class AssetComputeWorkerPipeline {
         return engine;
     }
 
+    normalizeInputOuput(input, output) {
+        console.log('input before normalization: ', input);
+        console.log('outpu before normalization: ', output);
+
+        if(output.fmt === 'machine-metadata-json') {
+            output.type = 'machine-metadata-json';
+        } else {
+            // rendition.fmt should always exist
+            const mimetype = mime.lookup(output.fmt.toLowerCase());
+            output.type = mimetype && mimetype.toLowerCase();
+        }
+
+        // source.mimetype will always exist, no need to cultivate here.
+        input.type = input.mimetype && input.mimetype.toLowerCase();
```

Comment:
what if source mimetype does `image/jpg` instead of `image/jpeg`? (or is this not a valid mimetype)
same goes for `image/tiff` vs `image/tif`

Reply:
if not already passed in, source gets it mimetype from request headers(in core). Can Manifest handle these synonyms, if not what does Manifest rely on 3 letter or 4 letter ? Can add a static mapping here too.

Reply:
looks like Manifest always look for 4 letter mime types i.e. `image/jpeg` & `image/tiff`

Reply:
`image/jpg` is not a valid `jpeg` mimetype (sometimes you'll see it because people build the mimetype using the extension). Same for tiff: `image/tif` is not a valid mimetype.

Reply:
got it, to be defensive have added a mapping layer for `tif` & `jpg` for source. We can choose to extend it or drop it later as we integrate.